### PR TITLE
Categories

### PR DESCRIPTION
**Pull request by irosenb**
Phran needs a way to categorize people who are volunteering for FFF. 

>We need to be able to notate or "mark" specific members who fall into staff and volunteer categories in the organization.  This will assist us in managing all members without having to refer to the various separate excel files we currently maintain for each group (one member may fall into more than one group, and may move from group to group at various times):

>- Board Members
- Certified Mediums
- Active Volunteers
- Trained Sitters
- Afterlife Discussion Group Facilitators

This pull request implements a category mechanism. Phran can assign a category to any person in the admin console, search the people in a category, and then download a CSV of the those people in that category.

### CODE REVIEW

**Comment by irosenb** [comment]
@dinshaw please review

**Comment by dinshaw** [comment]
Do you want thin merged? Please squash into one commit and add a well formed, and informative commit message : http://git-scm.com/book/ch5-2.html (search for 'imperative').

**Comment by irosenb** [comment]
@dinshaw thin merged?

**Comment by dinshaw** [comment]
@irosenb

**Comment by irosenb** [comment]
Sorry? That message didn't have any content.

**Comment by dinshaw** [comment]
read the comment above.

**Comment by irosenb** [comment]
Do you mean thin server? I don't see it anywhere…

**Comment by dinshaw** [comment]
it was  a typo. read the comment as it is now, address the issues mentioned.

**Comment by irosenb** [comment]
Okay there's one commit :+1:

**Comment by irosenb** [comment]
@dinshaw good to merge?

File: app/admin/users.rb [hunk 1]
```diff
@@ -100,12 +108,12 @@ def scoped_collection
...
         row :promotional_media_mp3 do |biz|
           if biz && biz.promotional_media_mp3
-            link_to biz.promotional_media_mp3.attached_file_name, biz.promotional_media_mp3.attachment.url
+            link_to biz.promotional_media_mp3.attachment_file_name, biz.promotional_media_mp3.attachment.url
           end
         end
         row :promotional_media_upload do |biz|
           if biz && biz.promotional_media_upload
-            link_to biz.promotional_media_upload.attached_file_name, biz.promotional_media_upload.attachment.url
+            link_to biz.promotional_media_upload.attachment_file_name, biz.promotional_media_upload.attachment.url
```
Comment (dinshaw) [inline app/admin/users.rb]: Didn't this stuff go in, in a previous commit?

**Comment by dinshaw** [comment]
Still need a thoughtful description of what this feature is all about, and how you chose to implement it .
Please follow the link i posted and read the article about commit messages.

**Comment by irosenb** [comment]
Okay added a description to the PR.

**Comment by irosenb** [comment]
@dinshaw how does that look?

**Pull request merged**